apps/server/src/modules/h5p-editor/contentStorage/contentStorage.spec.ts

```diff
@@ -155,11 +247,11 @@ describe('ContentStorage', () => {
 
 	describe('addContent', () => {
 		describe('WHEN content is created successfully', () => {
-			it('should return contentId', async () => {
+			it('should save content', async () => {
 				const { metadata, content, user } = setup();
-				const contentID = await service.addContent(metadata, content, user);
-				expect(contentID).toBeDefined();
-				expect(typeof contentID).toBe('string');
+				await service.addContent(metadata, content, user);
+
+				expect(contentRepo.createContent).toHaveBeenCalledWith(expect.objectContaining({ metadata, content }));
 			});
 		});
 		describe('WHEN contentId is empty', () => {
```

Comment:
It's not new but this test is wrong. Two reasons:
1) There is no error thrown by addContent. (This might have changed because of your changes.)
2) The test succeeds anyway because the catch-block is never reached and a test without an expect succeeds. Generally you can't test for an error like this. You need to do it like so: `await expect(() => service.addContent(metadata, content, user, emptyContentId)).rejects.toThrow();`